Examples/MAX32665/SPI_3Wire/main.c

```diff
@@ -0,0 +1,228 @@
+    printf("SPI SS Connection   : Connect P0.7(GPIO) - P0.16\n");
+    printf("SPI Data Connection : Connect P0.25 - P0.17\n\n");
+
+    /***** Configure master *****/
+    ret = MXC_GPIO_Config(&gpio_p0_7);
+    if (ret != E_NO_ERROR) {
+        printf("\nGPIO INITIALIZATION ERROR\n");
+        return E_FAIL;
+    }
+
+    MXC_GPIO_OutSet(gpio_p0_7.port, gpio_p0_7.mask);
+
+    ret = MXC_SPI_Init(SPI_MASTER, 1, 0, 1, 0, SPI_SPEED, MAP_A);
+    if (ret != E_NO_ERROR) {
+        printf("\nSPI MASTER INITIALIZATION ERROR\n");
+        return E_FAIL;
+    }
+
+    ret = MXC_SPI_SetDataSize(SPI_MASTER, DATA_SIZE);
+    if (ret != E_NO_ERROR) {
+        printf("\nSPI MASTER INITIALIZATION ERROR\n");
+        return E_FAIL;
+    }
+
+    ret = MXC_SPI_SetWidth(SPI_MASTER, SPI_WIDTH_3WIRE);
+    if (ret != E_NO_ERROR) {
+        printf("\nSPI MASTER INITIALIZATION ERROR\n");
+        return E_FAIL;
+    }
+
+    ret = MXC_SPI_SetMode(SPI_MASTER, SPI_MODE_0);
+    if (ret != E_NO_ERROR) {
+        printf("\nSPI MASTER INITIALIZATION ERROR\n");
+        return E_FAIL;
+    }
+
+    /***** Configure slave *****/
+    ret = MXC_GPIO_Config(&gpio_cfg_spi1_ss0);
+
+    if (ret != E_NO_ERROR) {
+        printf("\nGPIO INITIALIZATION ERROR\n");
+        return E_FAIL;
+    }
+    ret = MXC_SPI_Init(SPI_SLAVE, 0, 0, 1, 0, SPI_SPEED, MAP_A);
+    if (ret != E_NO_ERROR) {
+        printf("\nSPI SLAVE INITIALIZATION ERROR\n");
+        return E_FAIL;
+    }
+
+    ret = MXC_SPI_SetDataSize(SPI_SLAVE, DATA_SIZE);
+    if (ret != E_NO_ERROR) {
+        printf("\nSPI MASTER INITIALIZATION ERROR\n");
+        return E_FAIL;
+    }
+
+    ret = MXC_SPI_SetWidth(SPI_SLAVE, SPI_WIDTH_3WIRE);
+    if (ret != E_NO_ERROR) {
+        printf("\nSPI MASTER INITIALIZATION ERROR\n");
+        return E_FAIL;
+    }
+
+    ret = MXC_SPI_SetMode(SPI_SLAVE, SPI_MODE_0);
+    if (ret != E_NO_ERROR) {
+        printf("\nSPI MASTER INITIALIZATION ERROR\n");
+        return E_FAIL;
+    }
+
+    MXC_NVIC_SetVector(SPI_SLAVE_IRQ, SPI_Slave_IRQHandler);
+    NVIC_EnableIRQ(SPI_SLAVE_IRQ);
+
+    /***** Initialize Transaction Parameters *****/
+    master_req.spi = SPI_MASTER;
+    master_req.txData = (uint8_t *)master_tx;
+    master_req.rxData = (uint8_t *)master_rx;
+    master_req.txLen = 0;
+    master_req.rxLen = 0;
+    master_req.ssIdx = SPI_MASTER_SSIDX;
+    master_req.ssDeassert = 1;
+    master_req.txCnt = 0;
+    master_req.rxCnt = 0;
+    master_req.completeCB = NULL;
+
+    slave_req.spi = SPI_SLAVE;
+    slave_req.txData = (uint8_t *)slave_tx;
+    slave_req.rxData = (uint8_t *)slave_rx;
+    slave_req.txLen = 0;
+    slave_req.rxLen = 0;
+    slave_req.ssIdx = SPI_SLAVE_SSIDX;
+    slave_req.ssDeassert = 1;
+    slave_req.txCnt = 0;
+    slave_req.rxCnt = 0;
+    slave_req.completeCB = NULL;
+
+    while (1) {
+        for (int i = 0; i < DATA_LEN; i++) {
+            master_tx[i] = i;
+            slave_tx[i] = i + 500;
```

Comment:
integer overflow on `slave_tx` here - is this intentional?

Reply:
No, I did not notice. I will change it to "slave_tx[i] = i;". And commit again.
Thanks

Reply:
I updated.